CMakeLists.txt

```diff
@@ -28,11 +28,11 @@ target_include_directories(
   $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/cpptrace/cpptrace>
 )
 
-#target_compile_features(
-#  cpptrace
-#  PUBLIC
-#  cxx_std_11
-#)
+target_compile_features(
+  cpptrace
+  PUBLIC
+  cxx_std_11
+)
```

Comment:
Is it necessary to force this at the config level?

Reply:
Yes, because the compiler will try to compile as C++98 (or something older) by default and will fail to compile most of the code.

Reply:
Ok, sounds good. Because I'd like C++23's `<stacktrace>` to exist as a backend I'd like to (later) make the logic here something like
```cmake
if(NOT DEFINED CMAKE_CXX_STANDARD)
  get maximum supported cxx_standard
  if(max supported standard < 11)
    error
  endif()
  set standard to the max supported standard
endif()
```
Hopefully that's possible in cmake. I can take care of that after this is merged :)